src/Fabulous.Avalonia/Views/Controls/Separator.fs

```diff
@@ -0,0 +1,23 @@
+namespace Fabulous.Avalonia
+
+open Avalonia.Controls
+open Fabulous
+open Fabulous.StackAllocatedCollections.StackList
+
+type IFabSeparator = inherit IFabTemplatedControl
+
+module Separator =
+    let WidgetKey = Widgets.register<GridSplitter>()
```

Comment:
Should we call it a `GridSplitter` instead of a `Separator` to match the name in Avalonia?

Reply:
There is a Separator https://docs.avaloniaui.net/docs/controls/separator#reference . ?

Reply:
GridSplitter is a separate one?  https://docs.avaloniaui.net/docs/controls/gridsplitter#reference

Reply:
Yes, looks like it's 2 different controls:
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/Separator.cs
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/GridSplitter.cs

The GridSplitter seems to be movable while Separator is not.
For this PR, I think it's better to change to `Widgets.register<Separator>`

Reply:
Ohh yeah my bad 😅 it’s playing with both

Reply:
Done :)